library/proc_macro/src/bridge/closure.rs

```diff
@@ -17,6 +17,7 @@ pub(super) struct Closure<'a, A, R> {
 struct Env;
 
 impl<'a, A, R, F: FnMut(A) -> R> From<&'a mut F> for Closure<'a, A, R> {
+    /// Create a `Closure` from a function
```

Comment:
```suggestion
    /// Create a `Closure` from an `FnMut` eg.(function, `||` closure)
```
Since the signature is a big hard to read I think this is helpful